sim/src/test/scala/midasexamples/TutorialSuite.scala

```diff
@@ -187,6 +187,11 @@ class ParityF1Test extends TutorialSuite("Parity") {
   runTest("verilator", true)
   runTest("vcs", true)
 }
+
+class ParityVitisTest extends TutorialSuite("Parity", platformConfigs = classOf[DefaultVitisConfig].getSimpleName) {
```

Comment:
I haven't quite figured out how i'd like to add Vitis-specific tests in a DRY way. This does suffice to check that everything compiles and simulates fine.